app/views/announcement_mailer/seven_day_warning.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>A scheduled announcement will go out in 7 days</h1>
+
+<%= render "announcement_mailer/announcement" %>
+
+<br>
+<i><%= link_to "Update the announcement and share what your organization has been working on here", @announcement %></i>
```

Comment:
We should probably move or duplicate the link above the announcement render.

Double check the link is absoute and not relative